book/src/fork-checkpoints.md

```diff
@@ -0,0 +1,68 @@
+# Fork Checkpoints
+
+This design describes the software architecture for fork check pointing.  It addresses the following challenges:
+
+* Forks are created at every block.
```

Comment:
Are they?

Reply:
@garious   Every block is a voting point.  Every vote can be rolled back.  A new fork is created at every block.

Reply:
Something is only called a fork if there are two paths.  Maybe you're just missing the word "potentially".